aws-test/tests/aws_kinesis_firehose_delivery_stream/variables.tf

```diff
@@ -47,7 +47,7 @@ data "null_data_source" "resource" {
 }
 
 resource "aws_s3_bucket" "firehose_bucket" {
-  bucket = "tf-firehose-bucket"
+  bucket = var.resource_name
```

Comment:
can you please revert this changes? The changes have already been merged into main https://github.com/turbot/steampipe-plugin-aws/pull/771/files#diff-799a5785764b494e2c76e1a05193b7873bf3c92ef08c1f0c3c851abf061429cdL9